windows/devices/uwb/UwbDevice.cxx

```diff
@@ -38,12 +38,26 @@ UwbDevice::DeviceName() const noexcept
     return m_deviceName;
 }
 
-std::shared_ptr<uwb::UwbSession>
+std::shared_ptr<::uwb::UwbSession>
 UwbDevice::CreateSessionImpl(uint32_t sessionId, std::weak_ptr<::uwb::UwbSessionEventCallbacks> callbacks)
 {
     return std::make_shared<UwbSession>(sessionId, std::move(callbacks), this);
 }
 
+std::shared_ptr<::uwb::UwbSession>
+UwbDevice::ResolveSessionImpl([[maybe_unused]] uint32_t sessionId)
+{
+    // TODO: implement this
+    // One problem is that m_uwbDeviceConnector is of type
+    // IUwbDeviceDdiConnector, which prevents use of the session-based connector
+    // functionality. That is needed here to obtain information about the
+    // session in question. For example, we need to call
+    // IUwbSessionDdi::[SessionGetState, GetApplicationConfigurationParameters,
+    // SessionGetRangingCount]. So, we may need to change the type of
+    // m_uwbDeviceConnector to UwbConnector instead.
+    return nullptr;
```

Comment:
I think maybe you can do a dynamic cast here so that the UwbDevice.hxx interface still shows the m_uwbDeviceConnector as IUwbDeviceDdiConnector, or maybe it doesn't really matter since m_uwbDeviceConnector is private anyways

Reply:
We could. I just thought of another idea though, where we can store once instance of each interface. I think that would be cleaner.